package.json

```diff
@@ -121,6 +121,7 @@
     "@department-of-veterans-affairs/generator-vets-website": "^3.3.5",
     "@octokit/rest": "^16.43.1",
     "@sentry/browser": "^5.4.0",
+    "@testing-library/react": "^10.2.1",
```

Comment:
I just started using testing-library the other day and it's by far my favorite way to write tests now